RadarSDK/RadarUser.h

```diff
@@ -145,6 +146,16 @@ typedef NS_ENUM(NSInteger, RadarLocationSource);
  */
 @property (assign, nonatomic, readonly) BOOL proxy;
 
+/**
+ A boolean indicating whether the user's location is being mocked, such as in a simulation. May be `false` if Fraud is not enabled.
+ */
+@property (assign, nonatomic, readonly) BOOL mocked;
+
+/**
+ Learned fraud state for the user. May be `nil` if Fraud is not enabled.
+ */
+@property (nonnull, copy, nonatomic, readonly) RadarFraud *fraud;
```

Comment:
`nonnull` designation here is at odds with the comment about coming back `nil` - seems like this should be nullable

Reply:
The following commit removes the second half of that sentence, since the way the code is written, the `fraud` object will never be `null`.